dashboard/src/components/Dictionary/DictionaryValue.js

```diff
@@ -6,9 +6,13 @@ import Typography from "@material-ui/core/Typography";
 
 const styles = theme => ({
   root: {
-    display: "table-cell",
+    display: "grid",
     paddingLeft: theme.spacing.unit,
     userSelect: "text",
+    "& > code": {
```

Comment:
I think a more ideal option would be to either make this an option of the codeblock component or add a wrapper component that provides this functionality. In its current state it may complicate making future changes. For instance if we wanted to have an inline code code block (eg. "The command run was <code>exit 0</code>.") we wouldn't want the implicit margin.

Being a bit more explicit would give us a bit more power tailor the output appropriately.

Reply:
It appears using our custom `Code` component instead of `code` solves this issue.